contents/dotfiles/Dotfiles.md

```diff
@@ -0,0 +1,218 @@
+<frontmatter>
+  title: Introduction to Dotfiles
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Dotfiles
+
+**Author(s): [Tiu Wee Han](https://github.com/tiuweehan)**<br>
+
+### What are Dotfiles?
+Dotfiles are plain text configuration files on Unix like systems (e.g. MacOS, Linux, BSD). Dotfiles store settings of almost every application, service and tool running on your system. These files control the behavior of applications from boot to termination and everything in between.
+
+<pic src="dotfiles-logo.png" alt="Dotfiles Logo" width="45%">
+  
+  <sub>_Figure 1. Dotfiles Logo_ [(source)](https://www.twilio.com/blog/using-dotfiles-productivity-bootstrap-systems)</sub>
+
+</pic>
+
+Some common examples of dotfiles include:
+- `.profile`, `.bashrc`, `.zshrc`: scripts that run when a Shell is started. 
+- `.nanorc`, `.vimrc`, `.emacs.d`: dotfiles for configuring text editors like nano, vim and emacs respectively.
+- `.gitconfig`, `.gitignore`: dotfiles containing git configurations.
+- `.ssh/config`: dotfiles containing SSH configurations.
+
+### Why Dotfiles?
+1. **Dotfiles improve productivity.** Dotfiles allow users to tailor their environments to their needs and preferences, from general purpose configurations to project-specific commands. There are also many widely available libraries and plugins to choose and match, providing the most optimal tools to improve their productivity.
```

Comment:
These ordered bullet points seems disjointed with the benefits below. Perhaps can keep the bullet points, add /merge the benefits as part of the bullet points, while the features to be in a completely separate section AKA `Dotfiles features`? This is a suggestion tho.

Reply:
Comparing with other book chapters, I feel features could be under the first section `what is Dot file` or have a separate section.

Reply:
Thanks for your suggestions! I actually based my chapter on the Go chapter where the features are integrated with the benefits, because the specific benefits are tied to each feature. But good point, I'll look into improving the flow.

Reply:
I'll be removing the bullet points entirely since I'll be repeating the points in the later sections. Thanks anyway!